### PR TITLE
Fix some Firefox layout issues.

### PR DESCRIPTION
**Pull request by mrandre**
This tweaks a bug with firefox's rendering of md-card. Basically, there is a css prop that sets max height to 100%. Firefox interprets this as the height of the visible viewport rather than 'what is needed', or whatever chrome actually is. By setting this value to inherit, the height is automatically inherited. This works in chrome and firefox.

### CODE REVIEW

**Comment by mrandre** [comment]
Let me know if you'd like a different name for the css class.

File: src/client/mtna/archive-template.jade [hunk 1]
```diff
@@ -19,6 +19,7 @@ div.relative(role="main"
     md-card(
       ng-if="state.pre.length > 0"
       flex-gt-sm="90" flex-gt-md="80"
+      class="firefox-tweak"
```
Comment (DavidSouther) [inline src/client/mtna/archive-template.jade]: Prefer using `md-card.firefox-tweak` syntax for declaring classes.

Also, is this necessary for all md-card, or just for the record-viewer card?

Comment (mrandre) [inline src/client/mtna/archive-template.jade]: It's for md cards that have one of those flex props set. Particularly flex-gt-md

**Comment by stanparker** [comment]
LGTM.

**Pull request merged**